modules/virt-confirming-policy-updates-on-nodes.adoc

```diff
@@ -3,39 +3,39 @@
 // * virt/node_network/virt-configuring-node-network-policy
 
 [id="virt-confirming-policy-updates-on-nodes_{context}"]
-= Confirming Policy updates on nodes
+= Confirming policy updates on nodes
```

Comment:
This might need to be reworded now to be more specific

Reply:
+1